workspaces/src/network/variants.rs

```diff
@@ -72,6 +95,80 @@ where
         Ok(res)
     }
 
+    pub async fn generate_tla_credentials(&self) -> (AccountId, SecretKey) {
+        let id = crate::rpc::tool::random_account_id();
+        let sk = SecretKey::from_seed(KeyType::ED25519, DEV_ACCOUNT_SEED);
+        (id, sk)
+    }
+
+    /// Creates a top level developement account.
+    /// On sandbox network it has a balance of 100 Near.
+    /// If you need more Near for your tests in sandbox consider using `root_account()` method:
+    ///
+    /// # Examples
+    /// ```
+    /// use near_workspaces::{result::Result, Account, network::Sandbox, Worker};
+    /// fn get_account_with_lots_of_near(worker: &Worker<Sandbox>) -> Result<Account> {
+    ///     worker.root_account()
+    /// }
+    /// ```
+    ///
+    pub async fn dev_create_tla(&self) -> Result<Account> {
+        let (id, sk) = self.generate_tla_credentials().await;
+        let account = self.create_tla(id.clone(), sk).await?;
+        Ok(account.into_result()?)
+    }
+
+    pub async fn dev_deploy_tla(&self, wasm: &[u8]) -> Result<Contract> {
+        let (id, sk) = self.generate_tla_credentials().await;
+        let contract = self.create_tla_and_deploy(id.clone(), sk, wasm).await?;
+        Ok(contract.into_result()?)
+    }
+}
+
+impl<T> Worker<T>
+where
+    T: DevNetwork + SponsoredAccountCreator + 'static,
+{
```

Comment:
Do we still need `AllowDevAccountCreation`? I would implement `SponsortedAccountCreator` for testnet and sandbox networks (the implementations will be different, but the public interface can be the same) and call `DevNetwork = Network + SponsoredAccountCreator`, and then here it will be just `T: DevNetwork + 'static`